docs/SupportedCommands/Commands.md

```diff
@@ -0,0 +1,70 @@
+## Supported Commands
```

Comment:
I agree this will be very helpful, however I am not quite sure how to get this information 🤔 ?